play-media/src/components/Pages/AboutUsPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { ContentBlock as IContentBlock } from '../../interfaces/contentBlock';
+import { Location } from '../../interfaces/location';
+import ContentBlock from '../ContentBlock/ContentBlock';
+import { HeroBannerSection } from '../HeroBanner/HeroBannerSection';
+import LocationsGrid from '../LocationsGrid/LocationsGrid';
+
+export const AboutUsPage = ({
+  contentBlocks,
+  locations,
+}: {
+  contentBlocks: IContentBlock[];
+  locations: Location[];
+}) => {
+  // DEMO SCENARIO CUSTOMIZATION:
+  // Change these to the respective content ids in your instance
+  const CONTENT_IDS = {
+    ABOUT: 'YXcFXM4yp0aKvHu5S8tmOg',
+    HISTORY: 'o-IjG5JYuE6JNUVx1KejPA',
```

Comment:
This content ID will not work because the portal removes dashes from our master item IDs when copying them to the destination Content Hub ONE tenant.

Reply:
I fixed it by removing the dash in the serialized item and pushing it to the dev Content Hub ONE tenant. I deleted the old item with the dash in its ID.